### ISSUE TITLE
Duplicated error report

### ISSUE THREAD

**Issue by const-t** [tempesta-tech/tempesta#2227]
During working with configuration faced strange behavior, Tempesta prints error reports twice:
```
[13925.858895] [tempesta fw] Warning: Configuration parsing has failed. Clean up...
[13925.963633] [tempesta fw] exiting...
[13926.116951] [tdb] Shutdown Tempesta DB
[13946.807356] [tdb] Start Tempesta DB
[13946.828263] [tempesta fw] Initializing Tempesta FW kernel module...
[13947.015465] [tempesta fw] ERROR: don't know how to handle: 'cache_use_stale'
[13947.016620] [tempesta fw] ERROR: configuration parsing error:
                  5: cache_use_stale 4* 4*;
                     ^^^^^^^^^^^^^^^^^^^^^^
[13947.019105] [tempesta fw] Warning: Configuration parsing has failed. Clean up...
[13947.139416] [tempesta fw] ERROR: don't know how to handle: 'cache_use_stale'
[13947.140538] [tempesta fw] ERROR: configuration parsing error:
                  5: cache_use_stale 4* 4*;
                     ^^^^^^^^^^^^^^^^^^^^^^
[13947.142926] [tempesta fw] Warning: Configuration parsing has failed. Clean up...
[13947.251560] [tempesta fw] exiting...
[13947.405052] [tdb] Shutdown Tempesta DB

```

After some investigation I found that `fprintf` that used by `sysctl` utility calls `write` twice in case of `EINVAL` error.

```
openat(AT_FDCWD, "/proc/sys/net/tempesta/state", O_WRONLY|O_TRUNC) = 3
write(3, "start\n", 6)                  = -1 EINVAL (Invalid argument)
write(3, "start\n", 6)                  = -1 EINVAL (Invalid argument)
close(3)                                = 0
```

`echo "start" > /proc/sys/net/tempesta/state` has expected behavior.
```
newfstatat(1, "", {st_mode=S_IFREG|0644, st_size=0, ...}, AT_EMPTY_PATH) = 0
write(1, "start\n", 6)                  = -1 EINVAL (Invalid argument)
close(1)                                = 0
```

It's definitely not a problem of `sysctl`, it calls `fprintf` only once as expected, I checked this.

**Comment by ai-tmpst** [comment]
sysctl built from the procps repo doesn't produce duplicated errors, but I have not found any fundamental difference in the code. 
Think it's better to not use sysctl and write directly to /proc/sys/net/tempesta/state.

**Comment by krizhanovsky** [comment]
Fixed in #2244
